errors.js

```diff
@@ -242,6 +242,12 @@ Errors.InvalidHeaderTypeError = TypedError({
     name: null
 });
 
+Errors.InvalidInitHostPortError = TypedError({
+    type: 'tchannel.invalid-init-host-port',
+    message: 'invalid host:port string in init header; got: {hostPort}',
```

Comment:
high cardinality string, leave the field up to the logging of this error object imo